android/app/src/main/res/drawable/ic_noti_bookmark.xml

```diff
@@ -0,0 +1,7 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Ensure that you are using `srcCompat` for setting this drawable.
Else it will crash on old versions